app/src/main/java/io/github/droidkaigi/confsched/prefs/DefaultPrefsSchema.java

```diff
@@ -5,12 +5,12 @@
 
 @Table(name = "io.github.droidkaigi.confsched_preferences")
 public abstract class DefaultPrefsSchema {
```

Comment:
Yes, we can! I updated it in https://github.com/konifar/droidkaigi2016/pull/371/commits/40df785c23a42a3e6dd60be3340db082a52be7f9